pkg/api/const.go

```diff
@@ -28,8 +28,9 @@ const (
 	RHEL   Distro = "rhel"
 	CoreOS Distro = "coreos"
 	// Supported distros by OpenShift
-	OpenShift39RHEL Distro = "openshift39_rhel"
-	OpenShiftCentOS Distro = "openshift39_centos"
+	OpenShift39RHEL     Distro = "openshift39_rhel"
+	OpenShiftCentOS     Distro = "openshift39_centos"
+	OpenShiftLatestRHEL Distro = OpenShift39RHEL
```

Comment:
why?

Reply:
this follows unix alternatives pattern. if no distro is defined in the `openshift.json` it fill default to `OpenShiftLatestRHEL` which we need to update each time we release stable version. So if somebody will be using "manual" deployment using acs-engine cli (not web ui) we could enforce defaults stable.